### ISSUE TITLE
Calculate val loss

### ISSUE THREAD

**Issue by Zephyr69** [iscyy/ultralyticsPro#133]
This is some great stuff! However, I can see that validation losses are not calculated. I'd like to see those losses. would there be a convenient way to enable that?
